src/Changelogs/testing/StubModuleLearningObjectGateway.ts

```diff
@@ -0,0 +1,20 @@
+import { LearningObjectGateway } from '../LearningObjectGateway';
+import { Requester } from '../typings';
+import { LearningObjectSummary } from '../../shared/types';
+import { Stubs } from '../../tests/stubs';
+import { LearningObject } from '../../shared/entity';
+import { STUB_CHANGELOG_IDS } from './ChangelogStubs';
+
+export class StubModuleLearningObjectGateway extends LearningObjectGateway {
```

Comment:
Use implements not extends. `StubModuleLearningObjectGateway` isnt a sub-class
```suggestion
export class StubModuleLearningObjectGateway implements LearningObjectGateway {
```

Reply:
Also, couldn't we just rename to `StubLearningObjectGateway` why is it necessary to specify `Module`?

Reply:
I have seen other modules follow the naming convention of interface = <name>gateway and concretion = module<name>gateway. I was just trying to be consistent. Thoughts?